### ISSUE TITLE
HTML too dark

### ISSUE THREAD

**Issue by francisco-morales97** [wnkz/monoglow.nvim#10]
Hello, I hope you are good. I'm writing this because I love your colorscheme and I want it to be more awesome.

The problem that I'm facing right now is that the html tags and its attributes look too dark to be readable. Here's a screenshot.
![Captura desde 2024-11-18 18-16-59](https://github.com/user-attachments/assets/414411ac-abc1-489f-8f1c-9b203a8bb221)

I'm using angular so I have to work a lot with html. I wanted to solve this issue by overriding the highlights inside the `on_highlight` but since the highlights are called `@tag` and `@tag.attributes` the plugin gives me an error when trying to open neovim.

Could you please give it a little more contrast? Thanks in advance.

**Comment by wnkz** [comment]
You're right, it seems a bit too dark ; it's been a long time since I opened an html file 😅 .
I'll have a look at what I can do to improve ; in the meantime, you should be able to override Treesitter highlight groups with the same syntax as this 👇🏻 

https://github.com/wnkz/monoglow.nvim/blob/fe8b02d5be4ff30f721a9196e3003df6479e9f26/lua/monoglow/groups/treesitter.lua#L90-L92
